projects/core/src/base-core.module.ts

```diff
@@ -0,0 +1,37 @@
+import { ModuleWithProviders, NgModule } from '@angular/core';
+import { StateModule } from './state/state.module';
+import { ConfigInitializerModule } from './config/config-initializer/config-initializer.module';
+import { ConfigModule } from './config/config.module';
+import { ConfigValidatorModule } from './config/config-validator/config-validator.module';
+import { I18nModule } from './i18n/i18n.module';
+import { CmsModule } from './cms/cms.module';
+import { GlobalMessageModule } from './global-message/global-message.module';
+import { ProcessModule } from './process/process.module';
+import { FeaturesConfigModule } from './features-config/features-config.module';
+import { SiteContextModule } from './site-context/site-context.module';
+import { OccModule } from './occ/occ.module';
+import { MetaTagConfigModule } from './occ/config/meta-tag-config.module';
+
+@NgModule({
+  imports: [
+    StateModule.forRoot(),
+    ConfigModule.forRoot(),
+    ConfigInitializerModule.forRoot(),
+    ConfigValidatorModule.forRoot(),
+    I18nModule.forRoot(),
+    CmsModule.forRoot(),
+    GlobalMessageModule.forRoot(),
+    ProcessModule.forRoot(),
+    FeaturesConfigModule.forRoot(),
+    SiteContextModule.forRoot(), // should be imported after RouterModule.forRoot, because it overwrites UrlSerializer
+    OccModule.forRoot(),
```

Comment:
It looks like we will remove things from the `OccModule` with next libraries and making it breaking changes. Would move it to spartacus-setup

Reply:
fixed 👍